docker/README.md

```diff
@@ -0,0 +1,116 @@
+3. If you have a NVIDIA GPU you will also need to install `nvidia-docker2`. Installation instruction can be found [here](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
+
+## Building image
+
+To build the docker image simply run:
+```bash
+bash build_docker.sh
+```
+
+This script has a few options that can be used to specify various build variations. The options are:
+```bash
+Usage: ./build.sh [-r, --rebuild] [-v, --verbose] [-i, --image-name] [-d, --distro] [--smid]
+Build the epfl-lasa/iiwa_ros  image.
+Options:
+  -r, --rebuild            Rebuild with --no-cache option.
+  -v, --verbose            Display additional info upon build.
+  -i, --image-name         Defines the name given to the image beeing built. Default: epfl-lasa/iiwa_ros
+  -d, --distro             Can be \"noetic\" or \"melodic\", default: noetic
+  --smid                   Build with smid flags on. Flags are off by default.
+  -h, --help               Show this help message."
+```
+
+## Spinning container
+
+To run the container, use:
+```bash
+bash start_docker.sh interactive
+```
+
+This will open the container command prompt. The user is `ros`, and starts in a ros workspace which is already built and sourced. So you might for example directly input a command such as:
+```bash
+roslaunch iiwa_gazebo iiwa_gazebo.launch
+```
+
+### Modes
+
+`start_docker.sh` offers 3 modes. 
+
+- `interactive`: Opens a prompt connected to the container. Stops the container as soon as the prompt is closed
+- `server`: Starts the container in the background. The container will only be stopped when stopped explicitly or when `start_docker` is called again with interactive or server. (We always stop the previous docker before starting it again)
+- `connect`: Allows to connect to the container which was started with the server command. This opens a prompt to the container. You can have as many prompt connected to a server as you want.
+
+### Options
+
+In addition a few options can be used when using  `start_docker`:
+
+```bash
+Usage: ./start_dockers.sh [interactive | server | connect] [-i, --image] [-u, --user]
+Build the '${IMAGE_NAME}' image.
+Options:
+  interactive            Spin the image in the console
+  server                 Spin the image as an ssh server
+  connect                Connects to an active container
+  -i, --image            The name of the image to use to start the container
+  -u, --user             Specify the name of the login user. (optional)
+  -h, --help             Show this help message and the one from aica-docker
+  Additional arguments are passed to the aica-docker command.
+```
+
+## Technical notes
+
+- The docker builds on top of docker images from the [aica-docker/docker-images](https://github.com/aica-technology/docker-images). And the `start_docker.sh` script uses their set of command. This might not be the best approach but this works for now so it is good enough.
```

Comment:
You could provide the current hash such that people can check out the one that works with your scripts

Reply:
Yes, Docker hash is actually good too, but I was thinking of the current commit of the aica-docker scripts initially

Reply:
Oh yeah good call, makes more sense

Reply:
Alright not cleanest doc I've written but latest commit fixes that